src/wallet/deterministic.go

```diff
@@ -31,23 +31,19 @@ type Option func(w *Wallet)
 
 // NewWallet generates Deterministic Wallet
 // generates a random seed if seed is ""
-func NewWallet(wltName string, opts ...Option) Wallet {
-	//old seed generation
-	//seedRaw := cipher.SumSHA256(secp256k1.RandByte(64))
-	//seed := hex.EncodeToString(seedRaw[:])
-
+func NewWallet(wltName string, opts ...Option) (*Wallet, error) {
```

Comment:
We have lots of remaining code use panic directly, which will cause node can't gracefully shutdown, I will refactor that later.